sdk/go/common/apitype/core.go

```diff
@@ -357,6 +357,8 @@ type ResourceV3 struct {
 	IgnoreChanges []string `json:"ignoreChanges,omitempty" yaml:"ignoreChanges,omitempty"`
 	// ReplaceOnChanges is a list of properties that if changed trigger a replace.
 	ReplaceOnChanges []string `json:"replaceOnChanges,omitempty" yaml:"replaceOnChanges,omitempty"`
+	// RefreshBeforeUpdate indicates that the provider requires this resource to always be refreshed.
```

Comment:
I think at this point we don't need to mention providers -- it's true that right now that's the only way you can set this flag, but in future perhaps we'll find other neat use cases?
```suggestion
	// RefreshBeforeUpdate indicates that this resource should always be refreshed prior to updates.
```